File: WM.UI/src/app/base/models/Order.ts
export class Order {
    id!: number;
    sum!: number;
    statusId!: number;
    status!: string;
    facilityId!: number;
    facilityName!: string;
    items!: OrderItem[];
}

export class OrderItem {
    id!: number;
    name!: string;
    amount!: number;
    price!: number;
    orderId!: number;
    productId!: number;
}

File: WM.Core/WM.Core.Domain/Entities/Order.cs
namespace WM.Core.Domain.Entities;

public class Order
{
    public int Id { get; set; }

    public List<OrderItem> Items { get; set; }
    public int FacilityId { get; set; }
    public Facility Facility { get; set; }
    public int StatusId { get; set; }
    public OrderStatus Status { get; set; }
    public DateTime? ClosedAt { get; set; }
}
